### PR TITLE
SVGKit: Remove lumberjack from SVGkit on Quickart.

### PR DESCRIPTION
**Pull request by yisaschar**

### CODE REVIEW

**Comment by lior7** [comment]
@barakwei hi, if its ok, I will review it first, then you can give it to someone from Foundations,
we have to decide if we want to merge it to 2.x or give it a different name.

File: SVGKit-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -529,41 +500,10 @@
 		00E86D511741586600F0FA0A /* Vendor */ = {
```
Comment (giladkoch1) [inline SVGKit-iOS.xcodeproj/project.pbxproj]: Remove vendor folder as well

**Review by giladkoch1 — COMMENTED** [review]
F&S

File: SVGKit-iOS.xcodeproj/project.pbxproj (hunk 1 above)
Comment (yisaschar) [reply SVGKit-iOS.xcodeproj/project.pbxproj]: Done.

**Comment by yisaschar** [comment]
F&Sed.

**Comment by giladkoch1** [comment]
LGTM

**Pull request merged**